atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/PostgresConnectionConfig.java

```diff
@@ -79,7 +80,14 @@ public String getTestQuery() {
     @Value.Auxiliary
     public Properties getHikariProperties() {
         Properties props = new Properties();
-        props.putAll(getConnectionParameters());
+        Map<String, String> updatedConnectionParameters = new HashMap<>(getConnectionParameters());
+        // As per https://github.com/pgjdbc/pgjdbc/issues/1307, we need to replace ssl: true with sslmode: require
```

Comment:
Would also be helpful to link to the commit that changed the behavior:
https://github.com/pgjdbc/pgjdbc/commit/cdeeaca47dc3bc6f727c79a582c9e4123099526e